### ISSUE TITLE
False crash with reason "Unschedulable"

### ISSUE THREAD

**Issue by divomen** [abahmed/kwatch#318]
**Describe the bug**
When I increase number of replicas, kwatch send me notification in slack about crash with reason Unschedulable (befor autoscaler adds a node).
I tried to disable this reason by config:
```
namespaces:
  - default
  - staging
  - demo
reasons:
  - "!Unschedulable"
```
But it still sends notification with this reason

**To Reproduce**
just disable this reason in config and increase number of replicas of a pod until there is not enough resources in current nodes

**Expected behavior**
Not to send a notification with disabled reason

**Actual behavior**
send a notification with disabled reason

**Version/Commit**
0.9.3

**Comment by thomasLeclaire** [comment]
Hi! Could be linked or not but I'm also experiencing some abnormal behavior when we use some autoscaling. 
with v 0.9.3 I get some alerts on any scale down even if pods are normally and expectedly stopped.

**Comment by abahmed** [comment]
Hello @thomasLeclaire 
Can you provide more details (e.g. logs of kwatch) in a separate issue?

**Comment by abahmed** [comment]
@divomen Should be fixed in Release [v0.9.4](https://github.com/abahmed/kwatch/releases/tag/v0.9.4)

@thomasLeclaire Can you give a try with the new release? and if it still happening open an issue with details to be able to reproduce & fix it
